desktop/renderer-app/src/pages/SplashPage/index.tsx

```diff
@@ -79,9 +80,16 @@ export const SplashPage = observer<{}>(function SplashPage() {
                 className={classNames("splash-content", {
                     // @TODO add loading status
                     "is-success": loginStatus === LoginStatusType.Success,
+                    "is-logo-loaded": logoLoaded,
```

Comment:
From the classname `s-logo-loaded`. loading is already supported. Comments can be removed

Reply:
Logo loaded is not the same as login loading

Reply:
Logo loaded is for the updated logo animation.